Makefile

```diff
@@ -66,7 +66,7 @@ LEARNING_RATE := 0.002
 endif
 
 # Network specification. Default: $(NET_SPEC)
-NET_SPEC := [1,36,0,1 Ct3,3,16 Mp3,3 Lfys48 Lfx96 Lrx96 Lfx256 O1c\#\#\#]
```

Comment:
Since this is a wide-ranging change, can you do this as a separate PR pls?

Reply:
Done. https://github.com/tesseract-ocr/tesstrain/pull/216

Reply:
Also added separate PRs for other changes that can be handled separately.
https://github.com/tesseract-ocr/tesstrain/pull/214
https://github.com/tesseract-ocr/tesstrain/pull/215
https://github.com/tesseract-ocr/tesstrain/pull/217